wagtail/wagtailsearch/tests/test_elasticsearch5_backend.py

```diff
@@ -410,7 +410,7 @@ def test_none_lookup(self):
         # Check it
         expected_result = {'bool': {'filter': [
             {'match': {'content_type': 'searchtests.SearchTest'}},
-            {'missing': {'field': 'title_filter'}}
+            {'bool': {'mustNot': {'exists': {'field': 'title_filter'}}}}
```

Comment:
Fix here looks good to me, but it feels like we ought to have a test that actually runs the resulting query (even if it's just "throw this query at Elasticsearch and confirm that it doesn't break").